views/email_template.ejs

```diff
@@ -0,0 +1,120 @@
+<html>
+<head></head>
+<body>
+
+<table border="0" cellpadding="0" cellspacing="0" width="650">
```

Comment:
This should not be duplicated, at all. It needs to be in a separate template and included.